code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -525,6 +525,11 @@
 
 	var/damage = intensity - get_eye_protection()
 	if(.) // we've been flashed
+
+		var/datum/mutation/human/epilepsy/epilepsy_mutation = dna.get_mutation(/datum/mutation/human/epilepsy)
```

Comment:
would prefer if we instead made base living flash_act send a signal, that this mutation intercepts